### PR TITLE
feat: remove average and percent from table totals

### PR DESCRIPTION
**Pull request by rephus**
Closes: #2922

### Description:

Screenshot from 2022-08-15 16-12-08

### CODE REVIEW

**Review by IrakliJani — APPROVED** [review]
LGTM

File: packages/frontend/src/hooks/useColumnTotals.ts [hunk 1]
```diff
@@ -13,12 +16,28 @@ type Args = {
...
+const isSummable = (item: Field | TableCalculation) => {
+    if (isField(item) || isAdditionalMetric(item)) {
+        const numericTypes: string[] = [
+            DimensionType.NUMBER,
+            MetricType.NUMBER,
+            MetricType.COUNT,
+            MetricType.COUNT_DISTINCT,
+            MetricType.SUM,
+            MetricType.MIN,
+            MetricType.MAX,
```
Comment (ZeRego) [inline packages/frontend/src/hooks/useColumnTotals.ts:28]: We should remove min and max as well 🤔 what do you think ?

Comment (rephus) [reply packages/frontend/src/hooks/useColumnTotals.ts:28]: Mmmm, you're probably right. @TuringLovesDeathMetal can you think of a case where MIN/MAX can be useful on totals ?

Comment (TuringLovesDeathMetal) [reply packages/frontend/src/hooks/useColumnTotals.ts]: not really! Unless you're using min/max as a hack for getting a distinct value from a grouped dimension. 

let's remove it and then if someone complains/finds a use case for it, we can add it back :)

**Pull request merged**